internal/plugins/ansible/v1/init.go

```diff
@@ -213,20 +230,25 @@ func addInitCustomizations(projectName string) error {
 	if err != nil {
 		return err
 	}
-	err = util.ReplaceInFile(managerProxyPatchFile, "8081", "6789")
-	if err != nil {
-		return err
-	}
 
-	managerConfigFile := filepath.Join("config", "manager", "controller_manager_config.yaml")
-	err = util.ReplaceInFile(managerConfigFile, "8081", "6789")
-	if err != nil {
-		return err
+	if !componentConfig {
+		err = util.ReplaceInFile(managerProxyPatchFile, "8081", "6789")
+		if err != nil {
+			return err
+		}
 	}
-	// Remove the webhook option for the componentConfig since webhooks are not supported by ansible
-	err = util.ReplaceInFile(managerConfigFile, "webhook:\n  port: 9443", "")
-	if err != nil {
-		return err
+
+	if componentConfig {
+		managerConfigFile := filepath.Join("config", "manager", "controller_manager_config.yaml")
+		err = util.ReplaceInFile(managerConfigFile, "8081", "6789")
+		if err != nil {
+			return err
+		}
+		// Remove the webhook option for the componentConfig since webhooks are not supported by ansible
+		err = util.ReplaceInFile(managerConfigFile, "webhook:\n  port: 9443", "")
+		if err != nil {
+			return err
+		}
```

Comment:
Ideally we can condense this into the one if-else block above. I am fine with this going in like this because it preserves the same logic flow just wrapping it with conditionals. Just figured that I would mention that we should ideally come back and clean this up a bit :).

Reply:
updated in the if-else block.